internal/stacks/stackplan/planned_change_test.go

```diff
@@ -846,7 +846,7 @@ func TestPlannedChangeAsProto(t *testing.T) {
 				Before: cty.NullVal(cty.String),
 				After:  cty.StringVal("boop").Mark(marks.Ephemeral),
 			},
-			WantErr: "failed to encode raw state for var.thingy_id: can't serialize value marked with cty.NewValueMarks(marks.Ephemeral) (this is a bug in Terraform)", // Ephemeral values should never make it this far.
+			WantErr: "failed to encode after planned input variable var.thingy_id: : unhandled value marks cty.NewValueMarks(marks.Ephemeral) (this is a bug in Terraform)", // Ephemeral values should never make it this far.
```

Comment:
```suggestion
			WantErr: "failed to encode after planned input variable var.thingy_id: unhandled value marks cty.NewValueMarks(marks.Ephemeral) (this is a bug in Terraform)", // Ephemeral values should never make it this far.
```

I think there is a typo in the error message

Reply:
normally it would put the path to the broken attribute of the value in there, but because it's the whole value this is empty.

I was just going to leave it as is - we shouldn't see the error message at all (hopefully) and if we do it should be given straight to us and that part of the message is relevant still (ie. it's the whole value not some part).